recipe/run_test.sh

```diff
@@ -0,0 +1,15 @@
+nc-config --has-c++      | grep -q no
```

Comment:
Unless you specifically want to make sure that a feature is not present, I'd remove these negative tests.  I guess at best, it will force someone to change something when they add some new functionality, but at worst, people will be confused about why a package fails when they add some new feature.

I don't know if there's a happy middle ground here.  Perhaps adding these with "yes" for currently disabled options, but putting them behind a comment?

Reply:
You are right. I added those to test if the `grep` would work but I'll remove the `no` for now. The only one we actually want there in the future is the `--has-parallel`. The others are either separate libraries nowadays, like c++ and fortran, or we cannot package (`szlib`), or legacy (`pnetcdf`).